### DISCUSSION TITLE
gradle resource linking error

### DISCUSSION THREAD

**Discussion by Thazes** [patrykandpatrick/vico#898] [Questions]
### Question

I'm getting gradle resource linking error after i've imported vesrion 1.15 for views. The error is shown below
Android resource linking failed
ke.co.hfgroup.android.kCB-mergeDebugResources-84:/values/values.xml:8013: error: style attribute 'attr/gapLength (aka ke.co.hfgroup.android:attr/gapLength)' not found.
ke.co.hfgroup.android.kCB-mergeDebugResources-84:/values/values.xml:8021: error: style attribute 'attr/showGuidelines (aka ke.co.hfgroup.android:attr/showGuidelines)' not found.
ke.co.hfgroup.android.kCB-mergeDebugResources-84:/values/values.xml:8025: error: style attribute 'attr/positiveColor (aka ke.co.hfgroup.android:attr/positiveColor)' not found.
ke.co.hfgroup.android.kCB-mergeDebugResources-84:/values/values.xml:8026: error: style attribute 'attr/negativeColor (aka ke.co.hfgroup.android:attr/negativeColor)' not found.

Please help me sort it

### Vico version(s)

1.15

### UI framework(s)

Views

**Comment by patrickmichalik** [comment]
Hello! It looks like you’re attempting to use Vico 2 XML attributes with Vico 1. Please either upgrade to Vico 2 or use the Vico 1 attributes, which are listed [here](https://patrykandpatrick.com/vico/wiki/1.15.0/hosts#xml-attributes). It should be noted that split line styling is available only in Vico 2.

**Reply by Thazes** [reply]
Thank you for your Prompt response. Please clarify on the answer above because after rebuilding, I'm getting the sample errors below:

Android resource linking failed
ke.co.hfgroup.android.kCB-mergeDebugResources-84:/values/values.xml:8013: error: style attribute 'attr/gapLength (aka ke.co.hfgroup.android:attr/gapLength)' not found.
ke.co.hfgroup.android.kCB-mergeDebugResources-84:/values/values.xml:8021: error: style attribute 'attr/showGuidelines (aka ke.co.hfgroup.android:attr/showGuidelines)' not found.
ke.co.hfgroup.android.kCB-mergeDebugResources-84:/values/values.xml:8025: error: style attribute 'attr/positiveColor (aka ke.co.hfgroup.android:attr/positiveColor)' not found.
ke.co.hfgroup.android.kCB-mergeDebugResources-84:/values/values.xml:8026: error: style attribute 'attr/negativeColor (aka ke.co.hfgroup.android:attr/negativeColor)' not found.
ke.co.hfgroup.android.kCB-mergeDebugResources-84:/values/values.xml:8029: error: style attribute 'attr/line1Style (aka ke.co.hfgroup.android:attr/line1Style)' not found.
ke.co.hfgroup.android.kCB-mergeDebugResources-84:/values/values.xml:8037: error: style attribute 'attr/strokeThickness (aka ke.co.hfgroup.android:attr/strokeThickness)' not found.
ke.co.hfgroup.android.kCB-mergeDebugResources-84:/values/values.xml:8049: error: style attribute 'attr/verticalAxisItemCount (aka ke.co.hfgroup.android:attr/verticalAxisItemCount)' not found.
ke.co.hfgroup.android.kCB-mergeDebugResources-84:/values/values.xml:8051: error: style attribute 'attr/showTicks (aka ke.co.hfgroup.android:attr/showTicks)' not found.

**Reply by patrickmichalik** [reply]
Your project references `gapLength`, `showGuidelines`, and so on in some capacity. These are Vico 2 attributes—they don’t exist in version 1.15.0 and therefore cannot be used with it. Unless you upgrade to Vico 2, you’ll have to replace or remove these references.
